chalk-solve/src/clauses/builtin_traits/sized.rs

```diff
@@ -43,3 +34,53 @@ pub fn add_sized_program_clauses<I: Interner>(
     };
     builder.push_clause(trait_ref.clone(), iter::once(last_field_sized_goal));
 }
+
+fn handle_tuple<I: Interner>(
```

Comment:
Handle is a little generic, maybe `push_tuple_sized_conditions` or something like that?

Reply:
Same for `handle_struct`